We just need to pass an -L flag, or set lt_cv_deplibs_check_method=pass_all

--- origsrc/fftw-3.3.3/Makefile.am	2012-11-25 06:33:44.000000000 -0600
+++ src/fftw-3.3.3/Makefile.am	2013-11-27 14:24:39.278652500 -0600
@@ -81,13 +81,8 @@ libfftw3@PREC_SUFFIX@_la_LIBADD =			\
         $(SIMD_LIBS) $(SSE2_LIBS) $(AVX_LIBS) $(ALTIVEC_LIBS) $(NEON_LIBS)    	\
 	$(COMBINED_THREADLIBS)
 
-if QUAD
-# cannot use -no-undefined since dependent on libquadmath
-libfftw3@PREC_SUFFIX@_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
-else
 libfftw3@PREC_SUFFIX@_la_LDFLAGS = -no-undefined -version-info	\
 @SHARED_VERSION_INFO@
-endif
 
 fftw3@PREC_SUFFIX@.pc: fftw.pc
 	cp -f fftw.pc fftw3@PREC_SUFFIX@.pc
